web/src/main/resources/webapp/template/view/documents/details.html

```diff
@@ -1,18 +1,121 @@
 <!DOCTYPE html>
-<html xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout" xmlns:th="http://www.thymeleaf.org" layout:decorate="~{_layout/default}">
+<html xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout" xmlns:th="http://www.thymeleaf.org"
+	  layout:decorate="~{_layout/default}">
 <body>
 <th:block layout:fragment="content">
-    <section class="content-wrapper">
-        <section class="content-header">
-            <h1>[todo populate]</h1>
-        </section>
-        <form method="post" th:action="@{/documents/status/{id}(id=__${#request.getAttribute('id')}__)}">
-            <input type="submit" value="Change status" />
-        </form>
-        <form method="post" th:action="@{/documents/delete/{id}(id=__${#request.getAttribute('id')}__)}">
-            <input type="submit" value="Delete document" />
-        </form>
-    </section>
+	<section class="content-wrapper">
```

Comment:
Seems like somehow indentation has changed to 8 spaces in this file. Could you please revert to 4 spaces?

Reply:
Sure.